src/lib/tabs/tabs-common.scss

```diff
@@ -0,0 +1,51 @@
+@import '../core/style/variables';
```

Comment:
This file should either consist of exclusively mixins or exclusively concrete styles. 

If it's the former, then the file name should start with an underscore (which tells the sass compiler that it's a "partial" file that doesn't have output).

If it's the latter, it should be included in the "styleUrls" for each of the components that use it (rather than doing a sass `@import`)